app/models/cloud_volume.rb

```diff
@@ -18,6 +19,14 @@ class CloudVolume < ApplicationRecord
 
   acts_as_miq_taggable
 
+  supports :cloud_volume_backup_create do
+    unsupported_reason_add(:cloud_volume_backup_create, _("Backup unsupported")) unless validate_backup_create
```

Comment:
We want to ask also the class if it supports something and adding it here makes it return the wrong answer.
```ruby
Amazon::CloudVolume.supports_cloud_volume_backup_create? => true
```
Only if you ask instances in this case it would return the right answer.

So please remove this `supports` from the base class and add the `supports` to the child class. There you can also remove the `validate_backup_create` method of course.

Also renaming the feature being close to the actual method. `supports :backup_create` and `support :backup_restore`